src/e3/anod/spec.py

```diff
@@ -274,6 +274,17 @@ def enable_name_generator(self) -> bool:
         """
         return False
 
+    @property
+    def readme(self) -> tuple[str, str] | None:
```

Comment:
for me, readme_info is clearer because here the tuple return to information not linked:
  - the full path of the readme file (source)
  - the destination name of the readme in the store.